components/event_dispatcher.rst

```diff
@@ -189,10 +189,10 @@ determine which instance is passed.
 
 .. sidebar:: Registering Event Listeners and Subscribers in the Service Container
 
-    Registering service definitions and tagging them with the ``kernel.event_listener``
-    and ``kernel.event_subscriber`` tags is not enough to enable the event listeners
-    and event subscribers. You must also register the ``RegisterListenersPass()``
-    compiler pass in the container builder::
+    Registering service definitions and tagging them with the
+    ``kernel.event_listener`` and ``kernel.event_subscriber`` tags is not enough
+    to enable the event listeners and event subscribers. You must also register
+    in the container builder a compiler pass called ``RegisterListenersPass()``::
```

Comment:
You must also register a compiler pass call `RegisterListenersPass` in the container builder